libs/langchain/langchain/chat_models/base.py

```diff
@@ -121,7 +122,12 @@ def _convert_input(self, input: LanguageModelInput) -> PromptValue:
         elif isinstance(input, str):
             return StringPromptValue(text=input)
         elif isinstance(input, list):
-            return ChatPromptValue(messages=input)
+            try:
+                return ChatPromptValue(messages=input)
+            except BaseException as e:
+                raise ValidationError(
+                    f"There must be some element in the messages that is inappropraite for ChatPromptValue. "
+                ) from e
```

Comment:
does the original error not already make this clear?

Reply:
Yes, the original error message occurred from Pydantic and the error message is quite not clear.
The process in langchain only checked whethere the input is a list not checked the elements in the list are appropriate type. 

This is the error message I got

ValidationError: 1 validation error for ChatPromptValue
messages -> 6
  value is not a valid dict (type=type_error.dict)
  
  It is barely hard to understand why my input is wrong

Reply:
*P.S

I changed ValidationError into TypeError as original revised on occurred an error